app/assets/stylesheets/base/_variables.sass

```diff
@@ -81,3 +85,17 @@ $message-margin: $feed-item-margin
 $message-padding-vertical: $feed-item-padding-vertical
 $message-padding-horizontal: $feed-item-padding-horizontal
 $message-text-color: darken($application-main-color, 18%)
+
+// (invitation)form
+$form-margin: $feed-item-margin
+$form-padding: 35px
+$form-text-padding: 11px
+$form-background-color: $feed-item-background-color
```

Comment:
Is het misschien een idee om een globale `app-background-color` te maken en die zowel op de feed als op het invitation form (`.app` selector) toe te passen? Dat maakt wat duidelijker dat je altijd de inhoud van het vlak met de inhoud van de pagina een witte achtergrond wil geven.